enterprise/server/cmd/ci_runner/main.go

```diff
@@ -1158,10 +1158,15 @@ func (ar *actionRunner) Run(ctx context.Context, ws *workspace) error {
 		// completed so that the outer workflow invocation gets disconnected
 		// rather than finishing with an error.
 		if *workflowID != "" && exitCode == bazelLocalEnvironmentalErrorExitCode {
+			ar.reporter.Printf("Retryable error detected; killing ci_runner and retrying run...")
```

Comment:
During retries, don't we overwrite the invocation with the retry attempt? Wondering how helpful it will be to flush logs to an invocation page that is quickly inaccessible

Reply:
In the linked thread, it failed 5 times with the same error, so on the final retry attempt the invocation page would still be visible (in "disconnected" state) and we should be able to see these logs.

Reply:
(added a comment since this isn't obvious from looking at the code)

Reply:
Ahhh I see. Do you want to include the exit code in the message? In case that's helpful for users debugging

Reply:
We log the exit code a few lines up - `command exited with code %d`